src/components/DataSubmissions/DataSubmissionListFilters.tsx

```diff
@@ -0,0 +1,578 @@
+import { useOrganizationListContext } from "../Contexts/OrganizationListContext";
+import { ListSubmissionsInput, ListSubmissionsResp } from "../../graphql";
+import { useAuthContext } from "../Contexts/AuthContext";
+import { canViewOtherOrgRoles } from "../../config/AuthRoles";
+import { Column } from "../GenericTable";
+import { useSearchParamsContext } from "../Contexts/SearchParamsContext";
+import Tooltip from "../Tooltip";
+
+const StyledFilters = styled("div")({
+  paddingTop: "19px",
+  paddingBottom: "15px",
+  paddingLeft: "38px",
+  paddingRight: "26px",
+});
+
+const StyledFormControl = styled(FormControl)({
+  display: "flex",
+  flexDirection: "row",
+  alignItems: "center",
+  gap: "15px",
+});
+
+const StyledInlineLabel = styled("label")({
+  fontFamily: "'Nunito', 'Rubik', sans-serif !important",
+  fontWeight: 700,
+  fontSize: "16px",
+  lineHeight: "16px",
+  minWidth: "100px",
+  textAlign: "right",
+});
+
+const StyledSelect = styled(StyledSelectFormComponent)({
+  width: "280px",
+});
+
+const StyledTextField = styled(StyledTextFieldFormComponent)({
+  width: "280px",
+});
+
+const StyledRefreshIcon = styled(RefreshIcon)({
+  transform: "scale(-1, 1)",
+  color: "#346798",
+  fontSize: "31px",
+});
+
+const StyledIconButton = styled(IconButton)({
+  padding: 0,
+  border: "1px solid #D0D0D0",
+  borderRadius: "5px",
+});
+
+const StyledActionWrapper = styled(Box)({
+  minWidth: "44px",
+  width: "100%",
+  height: "44px",
+  display: "flex",
+  justifyContent: "flex-start",
+  alignItems: "center",
+});
+
+const ActionButtonsContainer = styled(Box)({
+  display: "flex",
+  flexDirection: "column",
+  justifyContent: "center",
+  height: "100%",
+  gap: "9px",
+});
+
+const initialTouchedFields: TouchedState = {
+  organization: false,
+  status: false,
+  dataCommons: false,
+  name: false,
+  dbGaPID: false,
+  submitterName: false,
+};
+
+const statusValues: SubmissionStatus[] = [
+  "New",
+  "In Progress",
+  "Submitted",
+  "Released",
+  "Withdrawn",
+  "Rejected",
+  "Completed",
+  "Canceled",
+  "Deleted",
+];
+
+const defaultValues: FilterForm = {
+  organization: "All",
+  status: "All",
+  dataCommons: "All",
+  name: "",
+  dbGaPID: "",
+  submitterName: "All",
+};
+
+type T = ListSubmissionsResp["listSubmissions"]["submissions"][0];
+
+export type FilterForm = Pick<
+  ListSubmissionsInput,
+  "organization" | "status" | "dataCommons" | "name" | "dbGaPID" | "submitterName"
+>;
+
+type FilterFormKey = keyof FilterForm;
+
+type TouchedState = { [K in FilterFormKey]: boolean };
+
+type ColumnVisibilityModel = { [key: string]: boolean };
+
+type Props = {
+  columns: Column<T>[];
+  submitterNames: string[];
+  dataCommons: string[];
+  columnVisibilityModel: ColumnVisibilityModel;
+  onColumnVisibilityModelChange: (model: ColumnVisibilityModel) => void;
+  onChange?: (data: FilterForm) => void;
+};
+
+const DataSubmissionListFilters = ({
```

Comment:
Thanks for catching this, it looks like it was just added yesterday when KC was updating the debounce time from 0.1s to 0.5s. I completely missed this new behavior. I went ahead and implemented it in [d0c03ae](https://github.com/CBIIT/crdc-datahub-ui/pull/476/commits/d0c03ae84aafdfe2d4d4b7b722352670a3af375f), [4b00bc5](https://github.com/CBIIT/crdc-datahub-ui/pull/476/commits/4b00bc5ce1f532aa67ad2c94583eb7e3eed32596), and [3322a2e](https://github.com/CBIIT/crdc-datahub-ui/pull/476/commits/3322a2e06e7c75ae5d6f2fa075343480dd8e6041).

I also prevented the searchParams from being updated until at least 3 characters are typed, that way if they refresh while under 3 characters it will avoid populating the field.